lib/alp_workloads/kvm_workload_utils.pm

```diff
@@ -58,6 +59,16 @@ sub set_kvm_container_image {
     }
 }
 
+sub set_host_bridge_interface_with_host {
+    # Setup Host Bridge Network via NetworkManager as needed
+    my $_alp_host_bridge = "/root/alp_host_bridge_init.sh";
+    assert_script_run("curl " . data_url("virt_autotest/alp_host_bridge_init.sh") . " -o $_alp_host_bridge");
+    assert_script_run("chmod +rx $_alp_host_bridge && $_alp_host_bridge");
```

Comment:
What if there is br0 already? Add the logic to skip?

Reply:
Good point! we have the same idea!  will add the basic check for the existed br0 situation.

Reply:
Add the basic check for the existed br0. If so, will skip to perform alp_host_bridge_init.sh script.

Reply:
Short update, add new function `check_activate_network_interface()` to check with br0 network interface.